benchmarks/arg_parser.py

```diff
@@ -24,6 +24,13 @@ def common_benchmark_parser():
             " depends on benchmark."
         ),
     )
+    parser.add_argument(
+        "--directory",
```

Comment:
Could we makes this --sample-directory? I think directory is too vague for this.